examples/userapi/07_sparse_operations.ipynb

```diff
@@ -0,0 +1,617 @@
+{
```

Comment:
<p>Add comment like: Some variable declarations</p><p>introduce diff between f and u (what is save)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2252/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>